cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -467,6 +469,11 @@ static long time(HttpServletRequest request) {
       return System.currentTimeMillis();
     }
 
+    if (startMs(request) != null || endMs(request) != null) {
+      throw new IllegalArgumentException("Parameter \"time\" and parameter \"start\"/\"end\" are mutually exclusive and "
```

Comment:
Nit: Rather than hard-coding the parameter names, can we use the corresponding variables? (e.g. via `String.format(...)`)